### PR TITLE
add the missing right bracket

### PR DESCRIPTION
**Pull request by yileic**

### CODE REVIEW

File: superset/assets/javascripts/explorev2/stores/controls.jsx [hunk 1]
```diff
@@ -991,7 +991,7 @@ export const controls = {
     description: 'Overlay a timeseries from a ' +
                  'relative time period. Expects relative time delta ' +
                  'in natural language (example:  24 hours, 7 days, ' +
-                 '56 weeks, 365 days',
+                 '56 weeks, 365 days)',
```
Comment (ascott) [inline superset/assets/javascripts/explorev2/stores/controls.jsx]: we don't need a bracket here, this is just a multiline string.

**Comment by xrmx** [comment]
Why this is not breaking the js build?

File: superset/assets/javascripts/explorev2/stores/controls.jsx (hunk 1 above)
Comment (yileic) [inline superset/assets/javascripts/explorev2/stores/controls.jsx]: sorry for my misunderstanding, I just saw there is a left one in line 993, thus thought so :P

Comment (ascott) [inline superset/assets/javascripts/explorev2/stores/controls.jsx]: oh i see, this is WITHIN the string and not part of the syntax, i missed that :p
thx for the fix :)

Comment (yileic) [inline superset/assets/javascripts/explorev2/stores/controls.jsx]: :)

**Pull request merged**